### DISCUSSION TITLE
pub get

### DISCUSSION THREAD

**Discussion by Amhaz1** [TheAlphamerc/flutter_twitter_clone#226] [Q&A]
Running "flutter pub get" in fwitter...
Because flutter_link_preview 1.5.6 depends on http ^0.12.1 and no versions of flutter_link_preview match >1.5.6 <2.0.0, flutter_link_preview ^1.5.6 requires http ^0.12.1.

So, because flutter_twitter_clone depends on both flutter_link_preview ^1.5.6 and http ^0.13.3, version solving failed.
pub get failed (1; So, because flutter_twitter_clone depends on both flutter_link_preview ^1.5.6 and http ^0.13.3, version solving failed.)
exit code 1
and am using flutter 2.0,5

**Comment by TheAlphamerc** [comment]
@Amhaz1 Can you take pull the latest code and try again?

**Comment by Amhaz1** [comment]
Okay

**Comment by Amhaz1** [comment]
![pub get error](https://user-images.githubusercontent.com/87565360/148658118-98c30bde-8466-4f52-b52c-bc6c68627379.png)

**Comment by Amhaz1** [comment]
am still getting the error please help me with solution
